apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedger.vm.ts

```diff
@@ -660,14 +663,22 @@ export class UpdateLedgerVm {
         this.discountArray = [];
     }
 
-    public calculateConversionRate(baseModel) {
+    /**
+     * Calculates conversion rate
+     *
+     * @param {*} baseModel Value to be converted
+     * @param {number} [customDecimalPaces] Optional custom decimal places (required for Rate as 4 digits are required for rate)
+     * @returns Converted rate
+     * @memberof UpdateLedgerVm
+     */
+    public calculateConversionRate(baseModel: any, customDecimalPaces?: number): number {
```

Comment:
Correct spelling of Places in customDecimalPlaces

Reply:
Done